dune-project

```diff
@@ -1,2 +1,26 @@
-(lang dune 1.0)
+(lang dune 1.11)
 (name pb)
+
+(generate_opam_files true)
+
+(implicit_transitive_deps false)
+
+(license MIT)
+
+(maintainers "Jeremy Yallop <yallop@gmail.com>")
```

Comment:
I think you should add yourself to `maintainers` (and to `authors`, too, if you like).